README.md

```diff
@@ -147,6 +147,10 @@ kubectl run jaeger-spark-dependencies --schedule="55 23 * * *" --env="STORAGE=ca
 
 If you want to run the job only once and immediately then remove scheduled flag.
 
+## Docker Versions
+The jaeger project automatically creates new docker images with versions that mirror the release number. Feel free to pick the latest release. 
+> A general k8 tip; it's recommended that you do not use `:latest` in production but rather pin the latest version.
```

Comment:
k8? Did you mean Kubernetes (k8s)? I think this tip is more general than that, so: 

> Docker pro-tip: do not use `:latest` in production, but rather, pin it to the latest release

Reply:
Correct the tags are parsed by docker, but seeing as this is a kubernetes repo and most people are deploying jaeger with kubernetes,  tagging this as a kubernetes/deployment related tip would be less confusing.